default.nix

```
@@ -1,7 +1,7 @@
 { system ? builtins.currentSystem, pkgs ? import ./nix { inherit system; } }:
 pkgs.buildGoApplication rec {
   pname = "cronosd";
-  version = "0.5.4-testnet";
+  version = "0.5.5-testnet";
```

```suggestion
  version = "0.5.5";
```

the nix built version don't have the `testnet` build tag.